cpp_test_suite/cpp_test_ds/DevTest.cpp

```diff
@@ -2612,8 +2612,14 @@ void DevTest::cmd_push_state_status_event()
 	set_change_event("State",true,false);
 	set_change_event("Status",true,false);
 
-    this->push_change_event("state", &get_state());
-    this->push_change_event("status", (Tango::DevString*)&get_status());
+	struct timeval tv;
+	gettimeofday(&tv, nullptr);
+
+	auto state = Tango::DevState::MOVING;
```

Comment:
@Ingvord Are we now always using C++11?

Reply:
No, this is just intermediate Progress commit. This PR will be cleaned up before labeled as `Ready For Merge`